### PR TITLE
Document new participant_join/leave events

### PR DESCRIPTION
**Pull request by martrik**
New events, YAY!

### CODE REVIEW

File: embed/index.html [hunk 1]
```diff
@@ -211,6 +211,16 @@ <h2>Listening to events</h2>
...
+    <td>{ participant: { metadata: String } }
+  </tr>
+  <tr>
+    <td><code>participant_leave</code>
+    <td>A participant leaves the room
+    <td>{ participant: { metadata: String } }
+  </tr>
 </table>
 
 <p>You can use standard Javascript to listen to the events. Here's a small demo:
```
Comment (dianaantunes) [inline embed/index.html:226]: should be added here in the example output and in the bottom listening to these new events if a room query param is provided

**Review by dianaantunes — CHANGES_REQUESTED** [review]
👍  but some things missing

**Comment by sj-e2digital** [comment]
I notice these events are snake_case but we also have "participantupdate" :/ I don't feel the name is very descriptive, could we deprecate it and add something more explanatory like "participant_count"?

File: embed/index.html (hunk 1 above)
Comment (martrik) [reply embed/index.html:226]: Oh well spotted, I didn't remember we had those examples underneath.

**Comment by martrik** [comment]
I agree @sj-e2digital , something for us to consider.

**Review by dianaantunes — APPROVED** [review]
LGTM

**Pull request merged**